src/actions/actionGroup.tsx

```diff
@@ -219,6 +215,7 @@ export const actionUngroup = register({
       { ...appState, selectedGroupIds: {} },
       getNonDeletedElements(nextElements),
       appState,
+      null,
```

Comment:
It seems that here we have access to the instance, why not sending it in?

Reply:
elements are not coming from state, so no point in tryin to retrieve from cache